<launch>
        <arg name="node_start_delay" default="50.0" />
        <param name="service_tag" type="str" value="pyTts"/>
        <param name="interface_type" type="str" value="voice_html" />
        <param name="html_service_tag" type="str" value="htmlServer"/>
        <node pkg="ros_pepper_pkg" name="rasa_action" type="rasa_action.sh" output="screen" args="src/nlu/" />
        <node pkg="ros_pepper_pkg" name="rasa_server" type="rasa_server.sh" output="screen" args="src/nlu/" />
        <node pkg="ros_pepper_pkg" type="nlu_interface.py" name="nlu_interface" output="screen" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' "/>
        <node pkg="ros_pepper_pkg" type="nlu_server.py" name="nlu_server" output="screen" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' "/>
        <node pkg="ros_pepper_pkg" type="voice_recorder.py" name="voice_recorder" output="screen" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' "/>
        <node pkg="ros_pepper_pkg" type="asr.py" name="asr" output="screen" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' "/>
        <node pkg="ros_pepper_pkg" type="tts_interface.py" name="tts_interface" output="screen" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' "/>
        <node pkg="ros_pepper_pkg" type="py_tts_server.py" name="py_tts" output="screen" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' "/>
        <node pkg="ros_pepper_pkg" name="html_server" type="html_server.sh" output="screen" args="src/html/" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' "/>
        <node pkg="ros_pepper_pkg" type="html_interface.py" name="html_interface" output="screen" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' "/>
        <node pkg="ros_pepper_pkg" type="html_server.py" name="wb_server" output="screen" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' "/>
</launch>